<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propAnnotationPoint</class>
 <widget class="QWidget" name="propAnnotationPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>321</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>9999999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignVCenter">
    <widget class="QGroupBox" name="annotationPoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>AnnotationPoint</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cxLabel">
        <property name="text">
         <string>X: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="ctrX">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cyLabel">
        <property name="text">
         <string>Y: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="ctrY">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 127, 0);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="czLabel">
        <property name="text">
         <string>Z: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="ctrZ">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>view as vector</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dirX">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cxLabel_2">
        <property name="text">
         <string>X: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="showNorm">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cyLabel_2">
        <property name="text">
         <string>Y: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="czLabel_2">
        <property name="text">
         <string>Z: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dirY">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 127, 0);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dirZ">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lenghtLabel">
        <property name="text">
         <string>Lenght:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="dirLen"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Show:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ctrX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedCtrX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedCtrY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctrZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedCtrZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedNormX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedNormY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedNormZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showNorm</sender>
   <signal>toggled(bool)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>toggledShowNorm(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirLen</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propAnnotationPoint</receiver>
   <slot>changedNormLen(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedCtrX(double)</slot>
  <slot>changedCtrY(double)</slot>
  <slot>changedCtrZ(double)</slot>
  <slot>changedNormX(double)</slot>
  <slot>changedNormY(double)</slot>
  <slot>changedNormZ(double)</slot>
  <slot>changedSize(int)</slot>
  <slot>toggledShowNorm(bool)</slot>
  <slot>changedNormLen(double)</slot>
 </slots>
</ui>
